plugins/inputs/tencentcloudcm/client.go

```diff
@@ -0,0 +1,102 @@
+package tencentcloudcm
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
+	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
+	monitor "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/monitor/v20180724"
+)
+
+type cmClient interface {
+	GetMetricObjects(t TencentCloudCM) []metricObject
+	NewClient(region string, crs *common.Credential, t TencentCloudCM) (monitor.Client, error)
+	NewGetMonitorDataRequest(namespace, metric string, instances []*monitor.Instance, t TencentCloudCM) *monitor.GetMonitorDataRequest
+	GatherMetrics(client monitor.Client, request *monitor.GetMonitorDataRequest, t TencentCloudCM) (*monitor.GetMonitorDataResponse, error)
+}
+
+type cloudmonitorClient struct {
+	Accounts []*Account
+	Log      telegraf.Logger `toml:"-"`
```

Comment:
One more superfluous TOML annotation here.

Reply:
removed in commit